src/cozmo/version.py

```diff
@@ -17,5 +17,5 @@
 # Specify the version of cozmoclad that this package requires
 # Releases of the Cozmo package must specify an exact cozmoclad release
 # to ensure compatibility with a specific release of the ios/android app.
-#__cozmoclad_version__ = None
-__cozmoclad_version__ = "1.0.1"
+__cozmoclad_version__ = None
```

Comment:
We should document this in README.md - I suspect that anyone forking this repo without following the cozmo installation steps, or even if they did follow those steps once, will run into issues - they'd need to know to manually install the correct cozmoclad module